arch/riscv/common/Kconfig

```diff
@@ -401,4 +401,10 @@ config PLIC
 	bool "Platform level interrupt controller (PLIC) support"
 	depends ARCH_HAS_PLIC
 
+config ARCH_HAS_I2C
+	bool
+
+config I2C
+	bool "Inter-Integrated Circuit (I2C) support"
```

Comment:
This has already been defined in kernel/i2c/Kconfig.

Reply:
done